app/helpers/spree/orders_helper.rb

```diff
@@ -17,17 +17,18 @@ def cart_count
 
     def changeable_orders
       # Only returns open order for the current user + shop + oc combo
-      return @changeable_orders unless @changeable_orders.nil?
-      return @changeable_orders = [] unless spree_current_user &&
-                                            current_distributor && current_order_cycle
-      return @changeable_orders = [] unless current_distributor.allow_order_changes?
-
-      @changeable_orders = Spree::Order.complete.where(
-        state: 'complete',
-        user_id: spree_current_user.id,
-        distributor_id: current_distributor.id,
-        order_cycle_id: current_order_cycle.id
-      )
+      @changeable_orders ||= if spree_current_user &&
+                                current_distributor&.allow_order_changes? && current_order_cycle
+
+                               Spree::Order.complete.where(
+                                 state: 'complete',
+                                 user_id: spree_current_user.id,
+                                 distributor_id: current_distributor.id,
+                                 order_cycle_id: current_order_cycle.id
+                               )
+                             else
+                               []
+                             end
```

Comment:
I can see how this is the equivalent to the above, but I'm not sure it makes it any easier to read. Was it in order to satisfy Rubocop? I would have probably just disabled the check on this block, but maybe that's just my personal opinion.

I think this is still reasonable 👍

Reply:
> Was it in order to satisfy Rubocop?

- I wanted to do it with the `begin...end` block but it gave the other Rubocop error something related to using the return keyword in the block.
- So I turned that block to this if condition. 
- Now I see that for better readability, this can be better moved into another private method 😅

Reply:
And David, I always get confused about whether I should disable Rubocop for some block or not 😅. For the future, can you please guide me on it? Thanks. ⭐

Reply:
No worries, I'm not always sure either! It's different for different people, which is one reason why we need Rubocop to keep things more consistent. So I think it's good that it now follows the rules.
I just like to question the rules sometimes ;)

Reply:
Makes sense. Thanks, David😄